src/app/services/user.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import {HttpClient} from "@angular/common/http";
+import {BehaviorSubject, first, Observable} from "rxjs";
+import {UserModel} from "../shared/user.model";
+import {JsonUsersStructureModel} from "../shared/json-users-structure.model";
+import {environment} from "../../environments/enviornment";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  userSubject: BehaviorSubject<UserModel[]> = new BehaviorSubject<UserModel[]>([]);
+  users$: Observable<UserModel[]> = this.userSubject.asObservable();
+  FILE_PATH: string = '../../assets/dummy-users.json';
```

Comment:
Don't insert users like this from the FE, just create a mongo script where you define all the users and just run the script in mongo shell